community/git.py

```diff
@@ -0,0 +1,61 @@
+import os
+import os.path
+
+from git.config import GitConfigParser
+import giturlparse
+
+REPO_DIR = os.path.join(
+    os.path.dirname(__file__),
+    '..',
+)
+GIT_CONFIG = os.path.join(
+    REPO_DIR,
+    '.git',
+    'config',
+)
+
+_config = None
+
+
+def get_config():
+    global _config
+    if not _config:
+        _config = GitConfigParser(GIT_CONFIG)
+    return _config
+
+
+def get_config_remote(name='origin'):
+    config = get_config()
+
+    print(config.sections())
```

Comment:
Is there a reason for this to be printing or is this just debug code?

Reply:
whoops!